### PR TITLE
remove redundant info in response when read single topic

### PR DESCRIPTION
**Pull request by MichaelGuoXY**
@jacgrady1 please review

### CODE REVIEW

File: services/topicsService.js [hunk 1]
```diff
@@ -70,7 +69,7 @@ export default {
                     { model: Answer, as: 'Answers', attributes: ['id', 'title', 'references', 'audio_url', 'updatedAt'], where: { 'status': { $not: StatusCode.DELETE }},
                         include: [
                             { model: User, as: 'User', attributes: ['id', 'username', 'user_role'] },
-                            { model: Like, as: 'Likes' },
+                            { model: Like, as: 'Likes', attributes: ['id', 'AnswerId', 'UserId'] },
```
Comment (MichaelGuoXY) [inline services/topicsService.js]: do not need "createAt" and "updateAt" attrs in Like

File: services/topicsService.js [hunk 2]
```diff
@@ -84,7 +83,6 @@ export default {
             }).then((topicInstance) => {
                 const instance = topicInstance.get({ plain: true });
                 data.topic = instance;
-                data.answers = instance.Answers;
```
Comment (MichaelGuoXY) [inline services/topicsService.js]: Answer duplicate

**Comment by MichaelGuoXY** [comment]
@jacgrady1 have tested locally, works properly

**Comment by jacgrady1** [comment]
Perfect. Was about to find the issue for you. But u found it!

**Pull request merged**